cmd/clusterctl/client/repository/repository_local.go

```diff
@@ -144,10 +144,14 @@ func newLocalRepository(providerConfig config.Provider, configVariablesClient co
 	// gets the path part of the url and check it is an absolute path
 	// in case of windows, we should take care of removing the additional / which is required by the URI standard
 	// for windows local paths. see https://blogs.msdn.microsoft.com/ie/2006/12/06/file-uris-in-windows/ for more details
-	path := url.EscapedPath()
+	path := url.Path
 	if runtime.GOOS == "windows" {
 		path = strings.TrimPrefix(path, "/")
 		path = filepath.FromSlash(path)
+	} else {
+		// Windows 10 update 1803 stops decoding file paths
+		// https://support.microsoft.com/en-us/help/4467268/url-encoded-unc-paths-not-url-decoded-in-windows-10-version-1803-later
+		path = url.EscapedPath()
```

Comment:
there could be a comment here on why the path is escaped for other OSes.

Reply:
Honestly...we could probably remove `url.EscapedPath()` altogether. Looking at the blame, it was introduced in this commit for adding Windows support: https://github.com/kubernetes-sigs/cluster-api/commit/737e36921121d12504bd78a66f8d86c72c5daba5

I don't believe escaping file paths is a requirement for other OSes. Thoughts?

Reply:
it was already present here before that commit:
https://github.com/kubernetes-sigs/cluster-api/commit/737e36921121d12504bd78a66f8d86c72c5daba5#diff-88fbf51cc189eba2daf622ffc4a215d0L134

Reply:
/cc @fabriziopandini 
do you recall why EscapedPath was required when preparing "basepath"?

Reply:
Ah, yup. Introduced here: https://github.com/kubernetes-sigs/cluster-api/commit/b5b9f79af96db9f186a2a0132044b91cd0f11473#diff-88bc45b90b0324f78a6bea9e5c59aa26R138

Reply:
TBH I don't recall.
I'm fine in removing it if tests report green signal on both OSes (unfortunately there is no CI signal for windows :-))

Reply:
Ok, I removed it. Manually tested on Windows and it works.